### DISCUSSION TITLE
Error while using anvil in test

### DISCUSSION THREAD

**Discussion by ghost** [foundry-rs/foundry#2857] [Help]
### Component

Anvil

### Have you ensured that all of these are up to date?

- [ ] Foundry
- [ ] Foundryup

### What version of Foundry are you on?

forge 0.2.0 (6cd6618 2022-08-11T00:03:54.64793456Z)

### What command(s) is the bug in?

cargo t -- --nocapture

### Operating System

Linux

### Describe the bug

I got this error while importing anvil for testing 
```
   Compiling foundry-config v0.2.0 (https://github.com/foundry-rs/foundry#6b3db2ad)
error[E0599]: no method named `include_paths` found for struct `ProjectBuilder` in the current scope
   --> /home/nxqd/.cargo/git/checkouts/foundry-f7cca724e93059b0/6b3db2a/config/src/lib.rs:602:14
    |
602 |             .include_paths(&self.include_paths)
    |              ^^^^^^^^^^^^^ method not found in `ProjectBuilder`

For more information about this error, try `rustc --explain E0599`.
error: could not compile `foundry-config` due to previous error
```

```
# Cargo.toml
anvil = { git = "https://github.com/foundry-rs/foundry" }
```

```
// usage in test
#[cfg(test)]
mod tests {
    use super::*;
    use anvil::{spawn, NodeConfig};

    #[tokio::test]
    async fn it_works() -> Result<()> {

        // create new provider with eth endpoint
        let (_api, handle) = spawn(NodeConfig::test()).await;

        Ok(())
    }
}

```

**Comment by mattsse** [comment]
you probably have a conflicting ethers dependency.

make sure that your lock file is up to date and perhaps run

`cargo update -p ethers`

and try bumping anvil itself

`cargo update -p anvil`

**Reply by ghost** [reply]
it works flawlessly, thanks.

**Comment by onbjerg** [comment]
Converted the issue to a discussion since it's not a bug on our end
